daemon/appinfo.c

```diff
@@ -59,6 +59,22 @@ typedef enum
     APPINFO_STATE_DELETED,
 } appinfo_state_t;
 
+typedef enum
+{
+    APPINFO_FILE_UNCHANGED,  // Nothing has changed
+    APPINFO_FILE_CHANGED,    // File timestamp has changed
+    APPINFO_FILE_INVALID,    // File is inaccessible => error
+    APPINFO_FILE_DELETED,    // File existed but was removed
+    APPINFO_FILE_MISSING,    // File never existed
+} appinfo_file_t;
+
+typedef enum
+{
+    APPINFO_DIR_MAIN = 0,
```

Comment:
nit: that zero looks a bit lonesome and useless (and the same goes for the other enum)

Reply:
I was thinking that better safe than sorry, since the value is used as array index it just has to be zero (but then I trusted that they are sequential <=> the next one is one). I guess the first item in enum is always 0 in C unless specified otherwise. I can drop these extra zeros.

Reply:
It is pretty much carved in stone since forever.

nb Draft C specs are available for free, and e.g. c99 says:

"6.7.2.2 Enumeration specifiers
... If the first enumerator has no =, the value of its enumeration constant is 0"

https://busybox.net/~landley/c99-draft.html#6.7.2.2